src/newLaunch/baseStage.scss

```diff
@@ -387,10 +387,25 @@
       }
 
       .pptContainer {
```

Comment:
are you sure that changing .pptContainer does not break the seed wizard layout?

Reply:
You are right. Though effect is minor, I revert this element in LBP to have a separate class.